src/WebAppDIRAC/Core/HandlerMgr.py

```diff
@@ -18,16 +10,13 @@
 from DIRAC.Core.Utilities.DIRACSingleton import DIRACSingleton
 from DIRAC.Core.Utilities.Extensions import extensionsByPriority, getExtensionMetadata
 
-import WebAppDIRAC
-
 from WebAppDIRAC.Lib import Conf
 from WebAppDIRAC.Lib.WebHandler import WebHandler, _WebHandler, WebSocketHandler
 from WebAppDIRAC.Core.CoreHandler import CoreHandler
 from WebAppDIRAC.Core.StaticHandler import StaticHandler
 
 
-@six.add_metaclass(DIRACSingleton)
-class HandlerMgr(object):
+class HandlerMgr:
```

Comment:
You lost the metaclass. Why?

Reply:
well noticed, forgot to type it in python 3 style